packages/shared/locales/pt-BR.json

```diff
@@ -528,6 +528,10 @@
                     "description": "A rede de incentivos trará grandes inovações para IOTA. O que quer que aconteça, acontecerá. Como o Shimmer evoluirá depende de você."
                 }
             }
+        },
```

Comment:
we should undo all the changes in this file, the CI and the community in crowdin will take care of the translation 🙏🏼